vmdb/app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -2,6 +2,7 @@
 
 module EmsRefresh::Parsers
   class Openstack < Cloud
+    include OpenstackMixin
```

Comment:
one request, could you place this under separate directory and do modules per service?

e.g. for orchestration_stacks I am doing
include EmsRefresh::Parsers::OpenstackCommon::OrchestrationStacks

so you would do 
include EmsRefresh::Parsers::OpenstackCommon::Flavors
include EmsRefresh::Parsers::OpenstackCommon::Images